apps/api/views/profiles.py

```diff
@@ -44,14 +45,30 @@ class TestStudentViewSet(ModelViewSet):
     serializer_class = serializers.profiles.TestStudentSerializer
 
 
+def make_ordinal(n):
+    '''
+    Convert an integer into its ordinal representation::
+
+        make_ordinal(0)   => '0th'
+        make_ordinal(3)   => '3rd'
+        make_ordinal(122) => '122nd'
+        make_ordinal(213) => '213th'
+    '''
+    n = int(n)
+    suffix = ['th', 'st', 'nd', 'rd', 'th'][min(n % 10, 4)]
+    if 11 <= (n % 100) <= 13:
```

Comment:
could write a few simple tests to cover this just-in-case

Reply:
✅